platform-core-module/src/main/java/com/softicar/platform/core/module/maintenance/status/AGMaintenanceStatus.java

```diff
@@ -0,0 +1,37 @@
+package com.softicar.platform.core.module.maintenance.status;
+
+import com.softicar.platform.common.core.entity.IEntity;
+import com.softicar.platform.common.core.exceptions.SofticarUnknownEnumConstantException;
+import com.softicar.platform.common.core.i18n.IDisplayString;
+import com.softicar.platform.core.module.CoreI18n;
+
+public class AGMaintenanceStatus extends AGMaintenanceStatusGenerated implements IEntity {
```

Comment:
as already suggested: I think the name `AGMaintenanceWindowState` would be preferable